.landscape.yml

```diff
@@ -10,4 +10,3 @@ ignore-patterns:
   - (^|/)docs(/|$)
 python-targets:
   - 3
-
```

Comment:
@vxgmichel do you know if this config file is still used ? I have the feeling we don't use this service since ages...

Reply:
The commit is from 6 years ago, I wasn't event here :)

Reply:
the file is removed